frontend/simple/views/Mailbox.vue

```diff
@@ -0,0 +1,284 @@
+                <div><strong style="margin-top: auto"><i18n>To</i18n>:</strong>&nbsp;
+                  <input id="AddRecipient" class="input is-small" type="text" style="width: 80%;" v-model="recipient" v-on:blur="addRecipient">
+                  <a class="button is-small" v-on:click="addRecipient">
+                    <span class="icon is-small">
+                      <i class="fa fa-plus-circle"></i>
+                    </span>
+                  </a>
+                </div>
+                <span v-if="error" id="badUsername" class="help is-danger"><i18n>Invalid Username</i18n></span>
+                <table>
+                  <tr v-for="(recipient, index) in recipients">
+                    <td>
+                      {{recipient.name}}
+                      <i class="fa fa-minus-circle icon is-small" v-on:click="removeRecipient(index)" ></i>
+                    </td>
+                  </tr>
+                </table>
+              </div>
+              <div class="panel-block">
+                <textarea id="ComposedMessage" class="textarea" v-model="composedMessage"></textarea>
+              </div>
+              <div class="panel-block" >
+                <button id="SendButton" class="button is-success" type="submit" v-on:click="send" :disabled="!composedMessage"  style="margin-left:auto; margin-right: 0"><i18n>Send</i18n></button>
+                <button class="button is-danger" type="submit" v-on:click="cancel" style="margin-left:10px; margin-right: 0"><i18n>Cancel</i18n></button>
+              </div>
+            </div>
+            <div id="CurrentMessage" class="panel" v-show="mode === 'Read'">
+              <div class="panel-heading">
+                <div><strong>Type:</strong>&nbsp;{{currentMessage.groupId ?  'Invite' : 'Message'}}</div>
+                <div><strong>Sent:</strong>&nbsp;{{formatDate(currentMessage.sentDate)}}</div>
+                <div><strong>From:</strong>&nbsp;{{currentMessage.groupId ?  currentMessage.groupId : currentMessage.from}}</div>
+              </div>
+              <p class="panel-block" v-if="currentMessage.message" style="display: block; word-wrap: break-word;">{{currentMessage.message}}</p>
+              <p class="panel-block" v-if="currentMessage.groupId"><router-link v-bind:to="{ path: '/join', query: { groupId: currentMessage.groupId} }" ><i18n>Respond to Invite</i18n></router-link></p>
+              <div class="panel-block" >
+                <button class="button is-danger" v-if="!currentMessage.groupId" type="submit" style="margin-left:auto; margin-right: 0" v-on:click="remove(index)"><i18n>Delete</i18n></button>
+                <button class="button is-primary" type="submit" v-on:click="inboxMode" style="margin-left:10px; margin-right: 0"><i18n>Return</i18n></button>
+              </div>
+            </div>
+            <table id="Invites" class="table is-bordered is-striped is-narrow"  v-show="(mode === 'Inbox')" v-if="invites.length">
+              <thead>
```

Comment:
Is `thead` / `tbody` used/needed? Does bulma take advantage of that? If not, it's not needed, just use `tr`s.